server/service/individualServices/ReadAirInterfaceData.js

```diff
@@ -1,31 +1,311 @@
+'use strict';
+
+/**
+ * @file This module provides functionality to gather the air-interface data for given mount-name and linkId. 
+ * @module readAirInterfaceData
+ **/
+const IndividualServiceUtility = require('./IndividualServiceUtility');
+const ltpStructureUtility = require('./LtpStructureUtility');
+const createHttpError = require('http-errors');
+
+const AIR_INTERFACE_LAYER_NAME = "air-interface-2-0:LAYER_PROTOCOL_NAME_TYPE_AIR_LAYER";
+const AIR_INTERFACE_CONFIGURATION = "air-interface-2-0:air-interface-configuration";
+const AIR_INTERFACE_CAPABILITY = "air-interface-2-0:air-interface-capability";
+const AIR_INTERFACE_STATUS = "air-interface-2-0:air-interface-status"
+
 var traceIndicatorIncrementer = 1;
 var mountName = "";
```

Comment:
I have a doubt. Please check the following scenario,
If multiple parallel request enters ALAP application to get the link acceptance details, keeping this variables global will create any impact ?

Reply:
removed global variables